src/app/utils/field-validation/validators/emailValidator.ts

```diff
@@ -0,0 +1,77 @@
+import { chain, left, right } from 'fp-ts/lib/Either'
+import { flow } from 'fp-ts/lib/function'
+import isEmail from 'validator/lib/isEmail'
+
+import { ProcessedSingleAnswerResponse } from 'src/app/modules/submission/submission.types'
+import { IEmailFieldSchema } from 'src/types/field'
+import { ResponseValidator } from 'src/types/field/utils/validation'
+
+import formsgSdk from '../../../../config/formsg-sdk'
+
+import { notEmptySingleAnswerResponse } from './common'
+
+type EmailValidator = ResponseValidator<ProcessedSingleAnswerResponse>
+type EmailValidatorConstructor = (
+  emailField: IEmailFieldSchema,
+) => EmailValidator
+
+const emailFormatValidator: EmailValidator = (response) => {
+  const { answer } = response
+  return isEmail(answer)
+    ? right(response)
+    : left(`EmailValidator:\t answer is not a valid email`)
+}
+
+const makeEmailSignatureValidator: EmailValidatorConstructor = (emailField) => (
```

Comment:
would it be possible to extract this logic so it can be shared between the mobile and email validators?